pkg/controller/upload-controller.go

```diff
@@ -526,6 +527,7 @@ func (r *UploadReconciler) makeUploadPodSpec(args UploadPodArgs, resourceRequire
 		Spec: v1.PodSpec{
 			SecurityContext: &v1.PodSecurityContext{
 				RunAsUser: &[]int64{0}[0],
+				FSGroup:   &fsGroup,
```

Comment:
Wondering if we should not set this for clone targets?  Assuming that cloning preserves ownership.  If not, it probably should, right?

And we should probably only be doing this for "kubevirt" `DataVolumeContentType`, right?

Reply:
You make very good points. Let me fix that.